prometheus/Readme.md

```diff
@@ -0,0 +1,62 @@
+# Prometheus Configuration
+
+## Default `query-max-concurrency` Setting: **4**
+
+The `query-max-concurrency` setting in Prometheus controls the maximum number of queries that can be executed concurrently.
+Setting this value appropriately is crucial to ensure that Prometheus can handle the load without overwhelming the system’s resources (CPU, memory, and disk).
```

Comment:
Mention that system instability can result if the memory resources are exceeded.

Clarify that for disk we are talking about IO operations, as opposed to disk space.